src/components/date_picker/super_date_picker/quick_select_popover/recently_used.tsx

```diff
@@ -59,18 +63,24 @@ export const EuiRecentlyUsed: FunctionComponent<EuiRecentlyUsedProps> = ({
   });
 
   return (
-    <Fragment>
+    <fieldset>
       <EuiTitle size="xxxs">
         <span>Recently used date ranges</span>
```

Comment:
We can open another ticket for this if you like but while you're in here, could you also swap this to a `legend` and i18n this string?

```suggestion
		<legend>
			<EuiI18n
	            token="euiRecentlyUsedDateRanges.legend"
	            default="Recently used date ranges"
	          />
	    </legend
```